### PR TITLE
Jc 8

### PR DESCRIPTION
**Pull request by aalexeen**

### CODE REVIEW

File: src/main/java/org/jacp/competitionservice/CompetitionserviceApplication.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.jacp.competitionservice;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class CompetitionserviceApplication {
```
Comment (saffchen) [inline src/main/java/org/jacp/competitionservice/CompetitionserviceApplication.java]: Please let's make CamelCase

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://${DB_HOST:localhost}:5432/competitionservice
+    username: competition
+    password: competition
+    driver-class-name: org.postgresql.Driver
+
+  liquibase:
+    change-log: classpath:src/main/resources/db/changelog/db.changelog-master.yaml
```
Comment (saffchen) [inline src/main/resources/application.yml]: Сan we specify the file path from the directory "resources"?

File: src/test/java/org/jacp/competitionservice/CompetitionserviceApplicationTests.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package org.jacp.competitionservice;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class CompetitionserviceApplicationTests {
```
Comment (saffchen) [inline src/test/java/org/jacp/competitionservice/CompetitionserviceApplicationTests.java:7]: Also CamelCase

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+                        </exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.liquibase</groupId>
+                <artifactId>liquibase-maven-plugin</artifactId>
+                <version>${liquibase-maven-plugin.version}</version>
+                <configuration>
+                    <changeLogFile>src/main/resources/db/changelog/db.changelog-master.yaml</changeLogFile>
```
Comment (saffchen) [inline pom.xml:78]: The configuration of liquibase is definitely needed in the pom.xml?

Comment (saffchen) [reply pom.xml:78]: Why is repetition necessary?

**Review by saffchen — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 4 above)
Comment (aalexeen) [reply pom.xml:78]: I can't see any repetition. This is a liquibase plugging configuration that allows to use Liquibase from Maven.

File: src/test/java/org/jacp/competitionservice/CompetitionserviceApplicationTests.java (hunk 3 above)
Comment (aalexeen) [reply src/test/java/org/jacp/competitionservice/CompetitionserviceApplicationTests.java:7]: done

File: src/main/resources/application.yml (hunk 2 above)
Comment (aalexeen) [reply src/main/resources/application.yml]: done

File: src/main/java/org/jacp/competitionservice/CompetitionserviceApplication.java (hunk 1 above)
Comment (aalexeen) [reply src/main/java/org/jacp/competitionservice/CompetitionserviceApplication.java]: done

File: pom.xml (hunk 4 above)
Comment (saffchen) [reply pom.xml:78]: Ok

**Review by saffchen — APPROVED** [review]

**Review by pelmegov — APPROVED** [review]

**Pull request merged**